docs/_pages/code-of-conduct.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Code of Conduct"
+permalink: /code-of-conduct/
+layout: splash
+header:
+  overlay_color: "#000"
+  overlay_filter: "0.5"
+  overlay_image: /assets/images/amc_banner.jpg
+#   actions:
+#     - label: "Download"
+#       url: "https://github.com/mmistakes/minimal-mistakes/"
+  caption: "Photo credit: [**buguroo**](https://www.buguroo.com/en/blog/topic/ai)"
+excerpt: "Conversational AI Club"
+toc: true
+toc_label: "Unique Title"
+toc_icon: "heart"  # corresponding Font Awesome icon name (without fa prefix)
+---
+
+## Code of Conduct
+
+### 1. Problem-driven
+
+> _"When you are being problem-driven, you approach product development and design not just based on specific features being requested of you, but by determining the underlying problems that your product is supposed to solve."_  
+> -- Leonid Batkhan, [Purpose-driven vs problem-driven vs solution-driven](http://www.lenetek.com/blog/purpose-driven-problem-driven-solution-driven/)
+
+### 2. Code first, paper second
+
+> _"Talk is cheap. Show me the code."_  
+> -- Torvalds, Linus (2000-08-25). [Message to linux-kernel mailing list.](https://lkml.org/lkml/2000/8/25/132)
+
+### 3. Always ask questions
+
+> ✔ _He who asks a question remains a fool for five minutes;_  
+> ✕ _He who does not ask remains a fool forever._
+
+### 4. Always follow the **two-pizza rule**
+
+> _"Never have a meeting where two pizzas couldn't feed the entire group."_  
+> -- Jeff Bezos, Amazon CEO
```

Comment:
I believe we had already move this file to the `manuals/` directory.

So this file should be created again.

Reply:
I have deleted duplicated files in my new commit.